src/sage/combinat/permutation.py

```diff
@@ -5270,49 +5270,52 @@ def shifted_shuffle(self, other):
         right_permutohedron_interval(self.shifted_concatenation(other, "left"))
 
 
-    def kth_roots(self, k=2):
+    def nth_roots(self, n):
         r"""
-        Return all k-th roots of self (as a generator).
+        Return all n-th roots of ``self`` (as a generator).
         
-        A k-th root of the permutation self is a permutation Gamma such that Gamma^k == self.
+        A n-th root of the permutation ``self`` is a permutation `\gamma` such that `\gamma^n == self`.
 
-        Note that the number of k-th roots only depend on the cyclic type of `self`.
-        
-        INPUT:
-        
-        - k -- optional integer (default 2), at least 1
+        Note that the number of n-th roots only depend on the cyclic type of ``self``.
```

Comment:
I think it is `cycle type`, rather than `cyclic type`.

Reply:
Indeed. I'm confuse by the French notation.